app/matplotlib.ipynb

```diff
@@ -152,30 +225,39 @@
     "* show it.\n",
     "\n",
     "> Please, note: you can get more information about histogram [here](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.hist.html)."
-   ],
-   "metadata": {
-    "collapsed": false
-   }
+   ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 8,
-   "outputs": [],
-   "source": [
-    "data = np.random.normal(0, 1, 1000)\n",
-    "\n",
-    "# write your code here\n"
-   ],
+   "cell_type": "raw",
    "metadata": {
-    "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-07T12:57:36.966131Z",
-     "start_time": "2023-07-07T12:57:36.923563Z"
+     "end_time": "2024-04-25T18:59:00.281379600Z",
+     "start_time": "2024-04-25T18:59:00.064219600Z"
+    },
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
     }
-   }
+   },
+   "source": [
+    "data = np.random.normal(0, 1, 1000)\n",
+    "\n",
+    "plt.hist(data, bins=50)\n",
+    "plt.title(\"Normal distribution\")\n",
+    "plt.xlabel(\"X\")\n",
+    "plt.ylabel(\"Y\")\n",
+    "\n",
+    "plt.show()"
+   ]
   },
   {
```

Comment:
Histogram is not shown
<img width="608" alt="image" src="https://github.com/mate-academy/py-matplotlib/assets/61380547/1760e647-34ae-481f-9bb1-9bcf53a5d6b7">